pkg/i18n/dutch.go

```diff
@@ -417,16 +417,16 @@ func addDutch(i18nObject *i18n.Bundle) error {
 			Other: `stage hunk`,
 		}, &i18n.Message{
 			ID:    "StageLine",
-			Other: `stage line`,
+			Other: `stage lijn`,
 		}, &i18n.Message{
 			ID:    "EscapeStaging",
-			Other: `return to files panel`,
+			Other: `ga terug naar het bestanden paneel`,
 		}, &i18n.Message{
 			ID:    "CantFindHunks",
-			Other: `Could not find any hunks in this patch`,
+			Other: `Kan geen hunks vinden in deze patch`,
 		}, &i18n.Message{
 			ID:    "CantFindHunk",
-			Other: `Could not find hunk`,
+			Other: `kan geen hunk vinden`,
```

Comment:
To keep structure the same `kan` should be capitalized.

Reply:
Thanks forgot to capitalize that, I'll change it!